catalog/harvest/catalog-harvest-common/src/main/java/org/codice/ddf/catalog/harvest/common/FileSystemPersistenceProvider.java

```diff
@@ -200,12 +210,21 @@ public boolean accept(File file, String name) {
     return keys;
   }
 
+  private String getPersistedFilePathForKey(String key) {
+    return getMapStorePath() + key + PERSISTED_FILE_SUFFIX;
+  }
+
   public void clear() {
     File[] files = new File(getMapStorePath()).listFiles(getFilenameFilter());
     if (files != null) {
       for (File file : files) {
-        if (!file.delete()) {
-          LOGGER.debug("File was unable to be deleted: {}", file.getAbsolutePath());
+        try {
+          Files.delete(file.toPath());
```

Comment:
Audits? You may also want to catch the AccessControlExceptions that can be thrown for `.listFiles` or `.delete`